docs/arcadia-users-group/overview.md

```diff
@@ -18,21 +18,21 @@ This is the landing page for tutorials delivered during AUG sessions.
 
 The schedule is subject to change.
 
-| Date | Tutorial | Description |
-|------|----------|-------------|
-| Oct 3, 2022   | [Code review using GitHub](https://arcadia-science.github.io/arcadia-computational-training/workshops/20220920-intro-to-git-and-github/lesson/#working-on-branches) | As the last piece of our [Git and GitHub workshop](../workshops/20220920-intro-to-git-and-github/lesson.md) we'll go over the key element of code review and how to execute a review on GitHub. |
-| Oct 10, 2022  | Indigenous peoples' day, no meeting | |
-| Oct 17, 2022  | Conda for software installation | In bioinformatics, we often use mulitiple software tools to answer a research question. In this lesson, we will learn how to use conda to install software. Conda is a software installation and management system that helps us conduct reproducible analyses while avoiding software conflicts. |
-| Oct 24, 2022  | Conda for environment management | In the previous lesson, we learned how conda works and how to install software. In this lesson, we'll learn how to organize our software installations into environments, and how to use environments to ensure our analyses are repeatable. |
-| Oct 31, 2022  | Turn a GitHub repo into a collection of interactive notebooks with Binder | This week we’ll introduce Binder, an executable environment that makes your code immediately reproducible by anyone, anywhere. |
+| Date         | Tutorial                                                                                                                                                            | Description                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Sorry for the all the stylistic changes, my linter just removed all the unnecessary spaces and made the table formats consistent 😬